corehq/apps/data_dictionary/templates/data_dictionary/base.html

```diff
@@ -34,27 +34,32 @@ <h2 class="text-hq-nav-header">{% trans "Data Dictionary" %}</h2>
     <div data-bind="saveButton: saveButton"></div>
     <div class="row">
         <div class="col-md-10">
-            <a class="btn btn-info" id="download-dict" href="{% url "export_data_dictionary" domain %}">Export to Excel</a>
-            <a class="btn btn-default" href="{% url "upload_data_dict" domain %}">Import from Excel</a>
+            <a class="btn btn-info" id="download-dict" href="{% url "export_data_dictionary" domain %}">{% trans "Export to Excel" %}</a>
```

Comment:
I mean expand/collapse all groups, not one at a time. The individual +/- seem fine. Don't think it's absolutely necessary, just wondering if you'd thought about it.

Documentation plan makes sense, how about just a link to that at the top? Would be good for us to have more direct connections between HQ and docs so people spend less time reading confluence search results.